### PR TITLE
Fix reselect character bug

### PR DESCRIPTION
**Pull request by shiweing**

### CODE REVIEW

File: schema/mvp.json [hunk 1]
```diff
@@ -36,8 +36,7 @@
     "apathy wildcard appeared": false,
     "sunny days wildcard appeared": false,
     "insomnia wildcard appeared": false,
-    "helping hand badge awarded": false,
-    "hyperfocus wildcard appeared": false
```
Comment (tlylt) [inline schema/mvp.json:40]: er is there a reason why this is removed? hyperfocus wildcard is for belinda I think.

File: schema/mvp.json [hunk 2]
```diff
@@ -9300,48 +9288,20 @@
...
                   "character": "Arthur",
                   "variant": "jacket",
-                  "text": "Hey Ethan, I'm sorry for not joining you guys, I wasn't up to it.",
-                  "characterDimension": {
-                    "width": "40%"
-                  },
-                  "characterPosition": {
-                    "right": "-13%",
-                    "bottom": "0%"
-                  }
```
Comment (tlylt) [inline schema/mvp.json:9310]: For all these, have you updated the image or? I think it was there to fix the dimension issues

**Review by tlylt — CHANGES_REQUESTED** [review]

**Comment by shiweing** [comment]
@tlylt I might have done something by mistake, I've added them back. Thanks for pointing that out

**Review by tlylt — APPROVED** [review]
LGTM

**Pull request merged**